src/reactions/aggregateReactions.js

```diff
@@ -1,19 +1,24 @@
 /* @flow strict-local */
-import type { Reaction, AggregatedReaction } from '../types';
+import type { Reaction, AggregatedReaction, User } from '../types';
+import { NULL_USER } from '../nullObjects';
 
 export default (
   reactions: $ReadOnlyArray<Reaction>,
   ownEmail: string,
+  users?: User[],
```

Comment:
If you really want to have the option of not passing users do it with a default value, like:
```js
users: User[] = []
```

Even better, instead of `getUsers` use the `getUsersById` selector and then  instead of `users.find` you can use `users[...]` to access the user.